### PR TITLE
Update kubevirt_vm with new feautures

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

This PR updates `kubevirt_vm` module, with latest changes.

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

kubevirt_vm

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py [hunk 1]
```diff
@@ -1,12 +1,9 @@
...
 # -*- coding: utf-8 -*-
 
-# Copyright: (c) 2019, Ansible Project
-# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
-
-from __future__ import absolute_import, division, print_function
-
-__metaclass__ = type
+# Copyright (c) 2018, KubeVirt Team <@kubevirt>
+# Apache License, Version 2.0
```
Comment (Akasurde) [inline lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py]: Not sure if we can license module code as Apache License.

Comment (Akasurde) [reply lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py]: cc @abadger @thaumos

Comment (machacekondra) [reply lib/ansible/modules/cloud/kubevirt/kubevirt_vm.py]: Ah, will change back. GPL

File: lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def list_items_by_resource_group(self):
         try:
             response = self.compute_client.disks.list_by_resource_group(resource_group_name=self.resource_group)
         except CloudError as exc:
-            self.fail('Failed to list items by resource group - {}'.format(str(exc)))
+            self.fail('Failed to list items by resource group - {0}'.format(str(exc)))
```
Comment (mattclay) [inline lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py]: This issue has been fixed in devel and does not need to be included in this PR.

Comment (machacekondra) [reply lib/ansible/modules/cloud/azure/azure_rm_managed_disk_facts.py]: rebased, thanks

**Review by pkliczewski — APPROVED** [review]

**Comment by pkliczewski** [comment]
shipit

**Review by mmazur — APPROVED** [review]
lgtm

**Comment by machacekondra** [comment]
@gundalow @Akasurde Can you please review & merge. Thank you!

**Pull request merged**